fn main() {
    // Exits the program right away with an error
    panic!();
}
